autoscheduler/frontend/src/package.json

```diff
@@ -20,6 +21,7 @@
     "@types/react-dom": "^16.9.1",
     "@typescript-eslint/eslint-plugin": "^2.4.0",
     "@typescript-eslint/parser": "^2.4.0",
+    "continuable-cache": "^0.3.1",
```

Comment:
Since `webpack-livereload-plugin` eventually ends up depending on `continuable-cache` (through `tiny-rl -> body -> continuable-cache`) which it has specified in the `package-lock.json`, shouldn't we not need to need to manually include this (assuming you did it to pass [this build failure](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/runs/305750606#step:5:8))?

Reply:
Theoretically, yes, that's how it should work, and in practice, it does work that way on my laptop, but not on GitHub Actions. I tried to see if I could replicate the problem in my own Docker container, but I got an `invalid volume specification .../postgres_username:/run/secrets/SECRET_POSTGRES_USER:ro` error. So at this point, it looks like we either just manually add it as a top-level dependency or you can try to run it without the top-level dependency in a clean Docker container and try to debug it yourself. My current theory is that Docker is using an older version of npm that ignores the package-lock and only looks at `package.json`

Reply:
So the Docker files we have inside of the repo are for the `Backend Testing` step, and aren't related to the other ones at all(which is why the error is related to Postgres). 

The step downloads Node(version 10.x) just runs `npm install`, so I'm really not sure why it's not correct. Likewise, I made a `frontend/live-reload-actionstest` branch just for messing with this and run [npm version](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/runs/306324728#step:5:1) on it, which has an npm version of 6.11.3, which is definitely up to date.

On that branch I reset to the `Merge branch 'frontend/master' into frontend/live-reload` commit before you added the `continuable-cache` package, ran `npm install` and committed the updated `package-lock.json` and it [passed the build](https://github.com/aggie-coding-club/Automatic-Aggie-Scheduler/runs/306324728#step:6:1), so I think updating the `package-lock.json`, rather than specifying the `continuable-cache` package directly should make it work.

Reply:
Oof, I was overthinking it. Fixed by removing the unnecessary dependency.